plugins/catalog-backend/config.d.ts

```diff
@@ -38,6 +38,33 @@ export interface Config {
        * E.g. ["Component", "API", "Template", "Location"]
        */
       allow: Array<string>;
+      /**
+       * Limit this rule to a specific location
+       *
+       * Example with a fixed location
+       *  { "type": "url", "exact": "https://github.com/a/b/blob/file.yaml"}
+       *
+       * Example using a Regex
+       *  { "type": "url", "pattern": "https://github.com/org/*\/blob/master/*.yaml"}
+       *
+       * Using both exact and pattern will result in an error starting the application
+       */
+      locations?: Array<{
+        /**
+         * The type of location, e.g. "url".
+         */
+        type: string;
+        /**
+         * The exact location, e.g.
+         * "https://github.com/org/repo/blob/master/users.yaml".
+         */
+        exact?: string;
```

Comment:
Is there a need to distinguish between `exact` and `pattern`? Seems like `exact` is a subset of `pattern` (a pattern without globs)

Reply:
It can be very handy if you want to match on locations that contain glob characters themselves. Also I think it's nice to convey the intent a bit more explicitly.

Reply:
Yes. I can add it to the documentation